org.freedesktop.Sdk.Extension.openjdk8.json

```diff
@@ -0,0 +1,213 @@
+{
+    "id": "org.freedesktop.Sdk.Extension.openjdk8",
+    "branch": "1.6",
```

Comment:
yes, it's your first remark of the PR ^^ because we want both, we can create branch/18.08 once the repo is created. Or the other way around, fine for me.

Reply:
Ah, makes sense. I'll merge it today.